protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -1227,18 +1225,18 @@ func renderServices(services []*descriptor.Service, paths openapiPathsObject, re
 							return err
 						}
 						if schema.Title != "" {
-							desc = mergeDescription(schema)
+							desc.WriteString(mergeDescription(schema))
```

Comment:
Isn't this actually incorrect? The old code overwrote `desc` but this appends to it.

Reply:
In this case desc with WriteString without allocate. If we use string - we have allocate in case like this s += "add text"

Reply:
I'm not saying I disagree with the use of WriteString, I'm just wondering if this is the equivalent logic, doesn't `WriteString` append, where this logic before would replace?

Reply:
Reverted this to source